provisioning/miniprov/hare_mp/main.py

```diff
@@ -529,6 +529,11 @@ def main():
                    help_str='Validates configuration pre-requisites',
                    handler_fn=noop)
 
+    add_subcommand(subparser,
+                   'upgrade',
+                   help_str='Upgrades the Hare RPM (Currently unavailable)',
```

Comment:
`help_str='Upgrades the Hare RPM....'`
I don't think we are upgrading RPM by using this interface. Are we?

Reply:
Right, let's reword it, how about just `Performs tasks to apply changes from the upgraded Hare rpm.`?

Reply:
Done!